scripts/combine/Templates/WMass/main.txt

```diff
@@ -2,7 +2,7 @@ imax 1
 jmax *
 kmax *
 ##----------------------------------
-shapes *  *  ${inputfile} ${histName}_$$PROCESS_$$CHANNEL ${histName}_$$PROCESS_$$SYSTEMATIC_$$CHANNEL
+shapes *  *  ${inputfile} $$PROCESS/${histName}_$$CHANNEL $$PROCESS/${histName}_$$SYSTEMATIC_$$CHANNEL
```

Comment:
I strongly believe the process name should also stay in the histogram name, otherwise it will be a mess to read them from root, we would need to update all names every time

Reply:
Yes histogram names should also be unique because there are global object lists...